### PR TITLE
Content build log clean up

### PR DESCRIPTION
**Pull request by indytechcook**
## Description

Content build log clean up.

## Testing done

Build finished successfully

image

Broken links always showing first

image

## QA steps

As user _uid_ with _user_role_
1. Do this
1. Then that
1. Then validate Acceptance Criteria from issue
- [ ] This
- [ ] That
- [ ] The other thing

### Definition of Done

- [ ] Documentation has been updated, if applicable.
- [ ] Tests have been added if necessary.
- [ ] Automated tests have passed.
- [ ] Code Quality Tests have passed.
- [ ] Acceptance Criteria in related issue are met.
- [ ] Manual Code Review Approved.

### Select Team for PR review

- [x] `Core Application Team`
- [ ] `Product Support Team`

### Is this PR blocked by another PR?

- [ ] `DO NOT MERGE`

### Does this PR need review from a Product Owner

- [ ] `Needs PO review`

### CMS user-facing annoucement

Is an announcement needed to let editors know of this change? 
- [ ] Yes, and it's written in issue ____ and queued for publication. 
  - [ ] Merge and ping @ rachel-kauff so she's ready to publish after deployment
- [ ] Yes, but it hasn't yet been written 
  - [ ] Don't merge yet -- ping @ rachel-kauff to prompt her to write and queue content
- [ ] No announcement is needed for this code change. 
  - [ ] Merge & carry on unburdened by announcements

### CODE REVIEW

File: docroot/modules/custom/va_gov_build_trigger/src/WebBuildBrokenLinkChecker.php [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+   *
+   * @var string
+   */
+  protected $summary;
+
+  /**
+   * Load Broken Links.
+   */
+  public function loadBrokenLinks() : void {
+    $path = $this->$this->getBrokenLinkPath();
```
Comment (ndouglas) [inline docroot/modules/custom/va_gov_build_trigger/src/WebBuildBrokenLinkChecker.php:39]: 🤔

File: docroot/modules/custom/va_gov_build_trigger/src/Plugin/VAGov/Environment/Tugboat.php [hunk 2]
```diff
@@ -68,8 +68,14 @@ public function triggerFrontendBuild(string $front_end_git_ref = NULL, bool $ful
...
 
+
+    // Remove broken links file just incaes it's still there
+    // from a previous build.
+    $commands = [
+      'rm -rf ' . $this->webBuildCommandBuilder->getAppRoot() . '/docroot/vendor/va-gov/web/logs/vagovdev-broken-links.json',
+    ];
     // A new command variable since the rebuild commands has been queued.
-    $commands = $this->webBuildCommandBuilder->buildCommands($front_end_git_ref);
+    $commands += $this->webBuildCommandBuilder->buildCommands($front_end_git_ref);
```
Comment (ndouglas) [inline docroot/modules/custom/va_gov_build_trigger/src/Plugin/VAGov/Environment/Tugboat.php:78]: I might be misremembering, but doesn't PHP's `+=` behave somewhat unintuitively with numerically indexed arrays?

For instance, if I evaluate the following:

```php
$array = [
  'rm stuff',
  'do something',
  'do another thing',
];
$array += [
  'do this last thing',
];
print_r($array);
```

I get 

```
Array
(
    [0] => rm stuff
    [1] => do something
    [2] => do another thing
)
```

not, as we'd expect

```
Array
(
    [0] => rm stuff
    [1] => do something
    [2] => do another thing
    [3] => do this last thing
)
```

We'd get that if we did `+=` with an associative array or if we used `array_merge()`, however.

I haven't tried your code, though, so IDK if I'm missing something or just being dumb.

File: docroot/modules/custom/va_gov_build_trigger/src/WebBuildBrokenLinkChecker.php (hunk 1 above)
Comment (indytechcook) [reply docroot/modules/custom/va_gov_build_trigger/src/WebBuildBrokenLinkChecker.php:39]: So many `$this` so little time...

File: docroot/modules/custom/va_gov_build_trigger/src/Plugin/VAGov/Environment/Tugboat.php (hunk 2 above)
Comment (indytechcook) [reply docroot/modules/custom/va_gov_build_trigger/src/Plugin/VAGov/Environment/Tugboat.php]: So this helped me led to another issue.  Thanks!

**Comment by ndouglas** [comment]
Encountered this error on my build: 

Screen Shot 2021-05-07 at 8 43 19 AM

I'm assuming that's not related to this PR and is Elijah rotating the GH token or something?

Otherwise, verified that the log's looking good.  Approving unless that error message is cause for concern.

**Review by ndouglas — APPROVED** [review]

**Comment by indytechcook** [comment]
@ndouglas was testing changing the PR but this was broken with the content build separation and will be fixed with #5194.  Normal builds work fine: 

image

**Pull request merged**